documentation/src/main/asciidoc/userguide/chapters/caching/Caching.adoc

```diff
@@ -807,24 +1021,34 @@ The old property still works, but its use is deprecated.
 Property `hibernate.cache.infinispan.use_synchronization` that allowed to register Infinispan as XA resource in the transaction has been deprecated in 5.0 and is not honored anymore. Infinispan 2LC must register as synchronizations on transactional caches. Also, non-transactional cache modes hook into the current JTA/JDBC transaction as synchronizations.
 ====
 
-[[caching-provider-infinispan-config-query-timestamps]]
-===== Configuring Query and Timestamps caches
+==== Handling custom identifiers types
 
-Since version 5.0 it is possible to configure query caches as _non-transactional_. Consistency guarantees are not changed and writes to the query cache should be faster.
+When custom identifier types are used in Hibernate/JPA entities, specially in
+the case of composite identifiers, the resulting cache keys can end up holding
```

Comment:
Sounds like it has been resolved in 5.0.4 (see [HHH-9424](https://hibernate.atlassian.net/browse/HHH-9424) and [HHH-10202](https://hibernate.atlassian.net/browse/HHH-10202)). I'll remove.